test/cppsuite/configs/reverse_split_stress.txt

```diff
@@ -0,0 +1,46 @@
+# This workload is inserts content continuously and truncates off the start of the file.
+# The truncations truncate up to 80% of the file each time.
```

Comment:
Should we explain how much the stress configuration differs from the default one? It's hard to go back and forth to compare the two and find the differences.

Reply:
87c1d8abf